src/services/copyConfig/consent/legalStatement.js

```diff
@@ -39,24 +38,40 @@ class LegalStatement {
     return res.data
   }
 
-  async copy(destinationSite, destinationSiteConfiguration, consentId, consentLanguages) {
+  async copy(destinationSite, destinationSiteConfiguration, consentId, consentLanguages, options) {
```

Comment:
This class do not need to be changed. See the comments on file ConsentConfiguration.